Tests/GettableSpec.swift

```diff
@@ -29,5 +52,43 @@ class GettableSpec: QuickSpec {
             
             itBehavesLike("a Restable") { context }
         }
+        
+        describe("A gettable with a dateDecodingStrategy") {            
+            it("It decodes the response data according to the dateDecodingStrategy") {
+                let gettable = DateRequest()
+                let currentDateString = "2018-06-12T20:00:00.000-04:00"
+                let currentDate = gettable.dateFormatter().date(from: currentDateString)!
```

Comment:
lines `59:60` are unnecessary - see comment below. However, given that we're going to remove the call to `try` then the `do/catch` is unnecessary.

Reply:
👍